sqldelight-gradle-plugin/src/test/fixtures/mapper-interfaces/expected/com/sample/Test1Model.java

```diff
@@ -42,31 +42,56 @@
     Test2Model test2();
   }
 
-  interface Join_tablesCreator<T extends Join_tablesModel> {
+  interface Join_tablesCreator<T extends Test1Model> {
     T create(Test1Model test1, Test2Model test2);
   }
 
-  final class Mapper<T extends Test1Model> implements RowMapper<T> {
-    private final Creator<T> creator;
+  final class Join_tablesMapper<T extends Test1Model, R0 extends Test1Model, R1 extends Test2Model> implements RowMapper<T> {
```

Comment:
Can we 1-index these type parameter names instead of 0-indexing? I think that's the common convention as far as I've found.